Assets/FullSerializer/Source/fsSerializer.cs

```diff
@@ -31,29 +31,59 @@ public class fsSerializer {
         /// <summary>
         /// This is an object reference in part of a cyclic graph.
         /// </summary>
-        private const string Key_ObjectReference = "$ref";
+        private static string Key_ObjectReference
+        {
+            get
+            {
+                return string.Format("{0}ref", fsGlobalConfig.InternalFieldPrefix);
```

Comment:
All of these should be computed once and then cached, otherwise they will cause quite a bit of memory allocation.